src/EluvioLive.js

```diff
@@ -2344,13 +2332,12 @@ class EluvioLive {
       // Generative NFT - build an nft array
 
       // Read image and attributes info from directory
-      let { nftMetas, rarity } = await this.readNftDir({ nftDir });
+      let { nftMetas } = await this.readNftDir({ nftDir });
```

Comment:
Can we still read this here in case we need it in the future?

Reply:
Yes, it's still there.